### PR TITLE
Rename project and update documentation

### PR DESCRIPTION
**Pull request by Jeremy-Walton**
## Task

Rename the project as we have agreed upon it

## What Changed

- [X] Update Repo Name
- [X] Update package name
- [X] Update Documentation and terminology used throughout.

## Sanity Check

- [X] Have you updated any usage of changed tokens?
- [X] Have you updated the docs with any component changes?
- [X] Have you run linters?
- [X] Have you run prettier?
- [X] Have you tried building the css?
- [X] Have you tried building storybook?
- [X] Do you need to update the package version?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,40 @@
-# Rolemodel Design System
+# Optics Design System
 
-RoleModel Design System is a scss package that provides base styles and components that can be integrated and customized in a variety of projects.
+Optics Design System by RoleModel Software is a scss package that provides base styles and components that can be integrated and customized in a variety of projects.
```
Comment (calebwoods) [inline README.md:3]:
```suggestion
Optics, a Design System by RoleModel Software is a scss package that provides base styles and components that can be integrated and customized in a variety of projects.
```

File: package.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "@rolemodel/rolemodel-design-system",
-  "version": "0.1.3-beta",
+  "name": "@rolemodel/optics",
+  "version": "0.2.0",
   "description": "The design system RoleModel uses to kickstart projects with a consistent starting point",
```
Comment (scriswell) [inline package.json:4]:
```suggestion
  "description": "Optics is an scss package that provides base styles and components that can be integrated and customized in a variety of projects.",
```

Comment (scriswell) [reply package.json:4]: @Jeremy-Walton Thinking this should be the same as what is stated within the `README.md`.

**Review by scriswell — APPROVED** [review]
@Jeremy-Walton Approved with one suggestion.

File: package.json (hunk 2 above)
Comment (Jeremy-Walton) [reply package.json]: Good call. Updated

**Pull request merged**